spec.html

```diff
@@ -299,7 +299,7 @@ <h1>CreateSegmentIterator ( _segmenter_, _string_ )</h1>
       <p>The CreateSegmentIterator abstract operation is called with arguments Intl.Segmenter instance _segmenter_ and String value _string_ to create a Segment Iterator over _string_ using the locale and options of _segmenter_. The following steps are taken:</p>
       <emu-alg>
         1. Let _internalSlotsList_ be &laquo; [[IteratingSegmenter]], [[IteratedString]], [[IteratedStringNextSegmentCodeUnitIndex]] &raquo;.
-        1. Set _iterator_ to ObjectCreate(%SegmentIteratorPrototype%, _internalSlotsList_).
+        1. Let _iterator_ be ObjectCreate(%SegmentIteratorPrototype%, _internalSlotsList_).
```

Comment:
I also noticed that in most places (except for 1.1.1 step 5a), we use `! ObjectCreate`. is it intentional that we don't use that here (or in 1.1.1 5a)?

Reply:
No, every use of ObjectCreate in this spec should be preceded by `!`. I'll land this PR and then fix that.